flang/runtime/command.cpp

```diff
@@ -282,4 +282,40 @@ std::int32_t RTNAME(GetEnvVariable)(const Descriptor &name,
   return StatOk;
 }
 
+const char *ensureNullTerminated(
```

Comment:
Capitalize the name, and put this function into `runtime/tools.cpp` or `runtime/memory.cpp`.

Reply:
moved to `tools`